packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewClientHostApiImpl.java

```diff
@@ -29,6 +33,8 @@ public class WebViewClientHostApiImpl implements GeneratedAndroidWebView.WebView
   private final WebViewClientCreator webViewClientCreator;
   private final WebViewClientFlutterApiImpl flutterApi;
 
+  private static HttpAuthHandler publicHttpHandler;
```

Comment:
Why is it called a publicHttpHandler? can't it be named something like httpAuthHandler. It is also not public but private?